_posts/2025-02-14-this-month-in-servo.md

```diff
@@ -0,0 +1,816 @@
+---
+layout:     post
+tags:       blog
+title:      "This month in Servo: and more!"
+date:       2025-02-14
+summary:    ao!! wrrrrao!!
+categories:
+---
+
+Servo now supports several new web API features:
+
+- **:host selector** (@simonwuelker, #34870)
+- **&lt;slot>** (@simonwuelker, #35013, #35177, #35191, #35221, #35137, #35222)
+- **Relative CSS colors**, like **color(from …)** and **rgb(from …)** (@Loirooriol, #34897)
+- **ClipboardEvent** (@Gae24, #33576, #35146), with `--pref dom_clipboardevent_enabled`
+- Detecting WGSL extensions via **WGSLLanguageFeatures** (@sagudev, #34928)
+- **pointer_composite_access** WGSL extension (@sagudev, #35161)
+- **blitFramebuffer()** on **WebGL2RenderingContext** (@jdm, Istvan, #26389)
+- **media** property on **HTMLStyleElement** (@webbeef, #35148)
+
+We’ve landed a bunch of **HTMLCanvasElement** improvements:
+
+- **toDataURL()** now supports **image/jpeg** and **image/webp** (@webbeef, #34861)
+- **toBlob()** is now supported (@webbeef, #34938)
+- **transferControlToOffscreen()** is now supported (@webbeef, #34959)
+
+**Streams** are a lot more useful now, with **ReadableStreamBYOBReader** now supporting **read()** (@Taym95, #35040), **cancel()**, **close()**, and **releaseLock()** (@Taym95, #34958).
+
+Servo now **passes [40.6%](https://wpt.servo.org)** (+7.5pp) of enabled **Shadow DOM** tests, thanks to our landing support for the **:host selector** (@simonwuelker, #34870) and the **&lt;slot> element** (@simonwuelker, #35013, #35177, #35191, #35221, #35137, #35222), plus improvements to event handling (@simonwuelker, #34788, #34884), script (@willypuzzle, #34787), style (@simonwuelker, @jdm, #35198, #35132), and the DOM tree (@simonwuelker, @Taym95, #34803, #34834, #34863, #34909, #35076).
+
+**Table layout** is significantly better now, particularly in ‘table-layout: fixed’ (@Loirooriol, #35170), table sizing (@Loirooriol, @mrobinson, #34889, #34947, #35167), rowspan sizing (@mrobinson, @Loirooriol, #35095), interaction with floats (@Loirooriol, #35207), and ‘border-collapse’ layout (@Loirooriol, #34932, #34908, #35097, #35122, #35165) and painting (@Loirooriol, #34933, #35003, #35100, #35075, #35129, #35163).
+
+As a result, Servo now **passes [90.2%](https://wpt.servo.org)** (+11.5pp) of enabled **CSS tables** tests, and of the tests that are in CSS 2, we now [**pass more than Blink and WebKit!**](https://wpt.fyi/results/css/CSS2?sha=d46d9abdea&max-count=1&to=2025-02-01T00%3A00%3A00.000Z&product=servo&product=chrome&product=safari&product=firefox)
```

Comment:
```suggestion
As a result, Servo now **passes [90.2%](https://wpt.servo.org)** (+11.5pp) of enabled **CSS tables** tests, and of the tests that are in CSS 2, we now [**pass more than Blink and WebKit!**](https://wpt.fyi/results/css/CSS2/tables?sha=d46d9abdea&max-count=1&to=2025-02-01T00%3A00%3A00.000Z&product=servo&product=chrome&product=safari&product=firefox)
```

Reply:
done in d3383d81e6de1f66fc5fc6a9466c8be8a18e7017